go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/gofrs/uuid/v5
+
+go 1.17
```

Comment:
Not necessary, but we could go ahead and roll this forward to 1.19.

Reply:
Updated!